docs/internal/investigations/runtime-governor-poc/README.md

```diff
@@ -0,0 +1,138 @@
+
+```bash
+watch kubectl --context=k3d-runtime1 get po --all-namespaces
+```
+
+In order to run the second Runtime cluster, please run all the previous steps using the `runtime2` scripts.
+
+### Use case scenario
+
+The setup described gives you the possibility to interact with two Runtime clusters. On both Runtimes, there is an example Dapr NodeApp deployed. It exposes two endpoints, called `/order` and `/neworder`, which interact with the Redis state store. You can call those endpoints using a Docker container running in the same Docker network as the clusters. 
+
+You can call the `/order` endpoint of Runtime 1 and Runtime 2 by running the following commands:
+
+```bash
+# Runtime 1
+docker run --network=cpnet --rm --entrypoint curl curlimages/curl -iv http://${$(docker inspect k3d-runtime1-server-0 --format='{{json .NetworkSettings.Networks.cpnet.IPAddress}}')//\"}/order
+
+# Runtime 2
+docker run --network=cpnet --rm --entrypoint curl curlimages/curl -iv http://${$(docker inspect k3d-runtime2-server-0 --format='{{json .NetworkSettings.Networks.cpnet.IPAddress}}')//\"}/order
+```
+
+In order to call the `/neworder` endpoint, send the **POST** request with the JSON in the body. See the example of such calls for Runtime 1 and Runtime 2:
+
+```bash
+# Runtime 1
+docker run --network=cpnet --rm --entrypoint curl curlimages/curl -iv http://${$(docker inspect k3d-runtime1-server-0 --format='{{json .NetworkSettings.Networks.cpnet.IPAddress}}')//\"}/neworder -d '{"data": {"orderId":"foo"}}' -H "Content-Type: application/json"
+
+# Runtime 2
+docker run --network=cpnet --rm --entrypoint curl curlimages/curl -iv http://${$(docker inspect k3d-runtime2-server-0 --format='{{json .NetworkSettings.Networks.cpnet.IPAddress}}')//\"}/neworder -d '{"data": {"orderId":"bar"}}' -H "Content-Type: application/json"
+```
+
+To prove that the Runtime configuration can be dynamically reloaded, change the configuration for Runtime 2 to point to the Redis 1 instance. To do so, run the following command:
+
+```bash
+REDIS1_PASSWORD=$(kubectl --context=k3d-cp get secret -n redis1-system redis -ojsonpath='{.data.redis-password}' | base64 -d) helm --kube-context=k3d-cp upgrade -i cp-poc -n cp-poc ./control-plane/chart \
+  --set redis1.host=${CP_IPADDR//\"}:30000 --set redis1.password=$REDIS1_PASSWORD \
+  --set redis2.host=${CP_IPADDR//\"}:30000 --set redis2.password=$REDIS1_PASSWORD
+```
+
+Now you can watch how the configuration is reloaded by the Runtime Agent on Runtime 2. The NodeApp Pod should be restarted.
+
+```bash
+watch kubectl --context=k3d-runtime2 get po --all-namespaces
+```
+
+### Cleanup
+
+To clean up the POC environment, use the following command:
+
+```bash
+./cleanup.sh
+```
+
+## Known issues
+
+Dapr sidecars need a restart to apply changes in configuration. We are still waiting for the decision on the Dapr side if they will stick to this approach.
```

Comment:
I'm not sure this information is up to date, as we received an update on that yesterday. I will provide you with the details through a DM.

Reply:
For the time of preparing this PoC it is up to date.

Reply:
Hmm... This POC is technically not finished yet until it's merged. I still think it's worth adding at least an `[EDIT:]` note to it. 
On the other hand, the point of the POC is to provide information to help make further decisions. If this information seriously affects the POC findings, is there a point merging such a POC containing outdated information?